src/jlapi.c

```diff
@@ -513,7 +513,8 @@ static int exec_program(char *program)
             jl_static_show(JL_STDERR, jl_current_exception());
             jl_printf(JL_STDERR, "\n");
         }
-        jlbacktrace();
+        jl_uv_flush(JL_STDERR);
+        jlbacktrace(); // written to STDERR_FILENO
```

Comment:
@vtjnash is the flush okay or should I rewrite the code above to use `STDERR_FILENO`?

Reply:
uv_flush is a yield point, and may cause your error to repeat ad nauseam